contracts/core/BasePaymaster.sol

```diff
@@ -104,7 +104,7 @@ abstract contract BasePaymaster is IPaymaster, Ownable {
 
     /// validate the call is made from a valid entrypoint
     function _requireFromEntryPoint() internal virtual {
-        require(msg.sender == address(entryPoint));
+        require(msg.sender == address(entryPoint), "Sender not EntryPoint");
```

Comment:
Do we want to have `AAxx` prefix on revert messages as well?

Reply:
no AAxxx prefixes are only added by entrypoint, not other contracts
(in fact, the EP  will add "AA23" prefix to mark it was thrown by the wallet)